eclipse-theia/theia
# vsx: use builtins published on open vsx

**Motivation**

https://github.com/theia-ide/vscode-builtin-extensions/issues/25#issuecomment-603147214
https://github.com/theia-ide/vscode-builtin-extensions/issues/25#issuecomment-603190961



#### What it does


This commit updates the examples to use the `builtins` published under Open VSX instead of Github releases.

_Update Exceptions:_
- node-debug
- node-debug2
- references-view

#### How to test


With `@theia/cli` built, running the script `yarn download:plugins` at the root of the repository should successfully download the plugins without any errors.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
